section-06-model-serving-api/house-prices-api/app/config.py

```diff
@@ -0,0 +1,70 @@
+import logging
+import sys
+from types import FrameType
+from typing import List, cast
+
+from loguru import logger
+from pydantic import AnyHttpUrl, BaseSettings
+
+
+class LoggingSettings(BaseSettings):
+    LOGGING_LEVEL: int = logging.INFO  # logging levels are type int
+
+
+class Settings(BaseSettings):
+    API_V1_STR: str = "/api/v1"
+
+    # Meta
+    logging: LoggingSettings = LoggingSettings()
+
+    # BACKEND_CORS_ORIGINS is a comma-separated list of origins
+    # e.g: http://localhost,http://localhost:4200,http://localhost:3000
+    BACKEND_CORS_ORIGINS: List[AnyHttpUrl] = [
+        "http://localhost:3000",  # type: ignore
+        "http://localhost:8000",  # type: ignore
+        "https://localhost:3000",  # type: ignore
+        "https://localhost:8000",  # type: ignore
+    ]
+
+    PROJECT_NAME: str = "House Price Prediction API"
+
+    class Config:
+        case_sensitive = True
+
+
+# See: https://loguru.readthedocs.io/en/stable/overview.html#entirely-compatible-with-standard-logging  # noqa
+class InterceptHandler(logging.Handler):
+    def emit(self, record: logging.LogRecord) -> None:  # pragma: no cover
+        # Get corresponding Loguru level if it exists
+        try:
+            level = logger.level(record.levelname).name
+        except ValueError:
+            level = str(record.levelno)
+
+        # Find caller from where originated the logged message
+        frame, depth = logging.currentframe(), 2
+        while frame.f_code.co_filename == logging.__file__:  # noqa: WPS609
```

Comment:
there is a lot of jargon here, like noqa, this code, pragma, meta. I have no idea what these are. Are you going to discuss them in the videos?
Otherwise, if they are there or not, makes no difference to me, for example.